server/views/partials/layout.njk

```diff
@@ -14,89 +14,96 @@
 {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
 {% from "../components/feedback-banner.njk" import feedbackBanner %}
 {% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "./breadCrumb.njk" import breadCrumb %}
 
 {% block head %}
 
-  {% if googleAnalyticsId %}
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="{{'https://www.googletagmanager.com/gtag/js?id=' + googleAnalyticsId}}"></script>
-    <script src="/assets/js/googleAnalytics.js" id='google-analytics' data-analytics={{googleAnalyticsId}}></script>
-  {% endif %}
+    {% if googleAnalyticsId %}
```

Comment:
Seems like 4 indents are our standard for `.njk` files? So thought I'd leave the auto-formatting changes in 💅